x11/auto-multiple-choice/Portfile

```diff
@@ -75,6 +75,7 @@ depends_run             \
                         port:p${perl5.major}-file-mimeinfo \
                         port:p${perl5.major}-glib-object-introspection \
                         port:p${perl5.major}-gtk3 \
+                        port:p${perl5.major}-locale-codes \
```

Comment:
Since you are adding this to `depends_run`, you need to increase the `revision` by 1 so that users who have it installed will get this new dependency installed.